cookies/templates/collection.html

```diff
@@ -35,6 +35,9 @@
     </span>
     <h2 class="collection_name">{{ collection.name }}</h1>
     <div class="collection_uri"><strong>URI:</strong> {{ collection.uri }}</div>
+    {% if collection.part_of != null%}
```

Comment:
The ``!= null`` bit is overkill -- if ``part_of`` is set it will be truthy, so ``{% if collection.part_of %}`` is sufficient.